dot/network/gossip_test.go

```diff
@@ -84,7 +84,9 @@ func TestGossip(t *testing.T) {
 		Digest: types.NewDigest(),
 	}
 
-	_, err = nodeA.host.send(addrInfoB.ID, "", announceMessage)
+	time.Sleep(TestMessageTimeout)
+
+	_, err = nodeA.host.send(addrInfoB.ID, "/gossamer/test/0/block-announces/1", announceMessage)
```

Comment:
Why do we need a sleep now?

Reply:
not sure. I'll take a look

Reply:
this test doesn't work.  I must've just left that line in trying to debug it.